projects/core/src/auth/store/reducers/user-token.reducer.ts

```diff
@@ -20,11 +20,9 @@ export function reducer(
 
     case fromAction.LOAD_USER_TOKEN_SUCCESS:
     case fromAction.REFRESH_USER_TOKEN_SUCCESS: {
-      const token = action.payload;
-
       return {
         ...state,
-        token,
+        ...action.payload,
```

Comment:
Please be careful with this merging syntax, it will merge unknown properties from payload.